Tools/MiniBrowser/gtk/main.c

```diff
@@ -972,6 +974,77 @@ static void activate(GApplication *application, WebKitSettings *webkitSettings)
     gtk_widget_show(GTK_WIDGET(mainWindow));
 }
 
+gboolean webkit_settings_apply_from_key_file(WebKitSettings *settings, GKeyFile *key_file, const char *group, GError ** error)
```

Comment:
```suggestion
gboolean webkit_settings_apply_from_key_file(WebKitSettings *settings, GKeyFile *key_file, const char *group, GError **error)
```

Reply:
Maybe it makes sense to add this to the api? load/save methods to GKeyFile/path

Reply:
I'm not sure about save, but yes load at least would be a good start.